src/index.js

```diff
@@ -0,0 +1,74 @@
+import '../style/style.scss';
+
+const form = document.querySelector('form');
+
+const myCity = document.getElementById('city-input');
+const btnGetWeather = document.getElementById('get-weather');
+const result = document.getElementById('result');
+const resultInner = document.createElement('div');
+resultInner.id = 'inner-content';
+resultInner.classList.add('text-center', 'mt-2', 'text-white');
+const temp = document.createElement('span');
+const tempUnit = document.createElement('h6');
+const h3 = document.createElement('h3');
+const p = document.createElement('p');
+const p1 = document.createElement('p');
+const p2 = document.createElement('p');
+const p3 = document.createElement('p');
+temp.style.fontSize = '6rem';
+temp.style.position = 'relative';
+tempUnit.style.position = 'absolute';
+tempUnit.style.top = '15px';
+tempUnit.style.right = '-10px';
+
+let url;
+let city;
+
+function reset() {
+  form.reset();
+}
+
+function displayRemove() {
+  resultInner.innerHTML = '';
+}
+
+async function getData(url) {
+  const response = await fetch(url, { mode: 'cors' });
+  const data = await response.json();
+  h3.innerHTML = `${data.name}`;
+  resultInner.appendChild(h3);
+  temp.innerHTML = Math.round(data.main.temp);
+  tempUnit.innerHTML = 'C';
+  p.innerHTML = `Feels Like : ${Math.round(data.main.feels_like)}`;
+  p1.innerHTML = `Minimum Temperature : ${Math.round(data.main.temp_min)}`;
+  p2.innerHTML = `Maximum Temperature : ${Math.round(data.main.temp_max)}`;
+  p3.innerHTML = `Humidity : ${Math.round(data.main.humidity)}`;
+  temp.appendChild(tempUnit);
+  resultInner.appendChild(temp);
+  resultInner.appendChild(p);
+  resultInner.appendChild(p1);
+  resultInner.appendChild(p2);
+  resultInner.appendChild(p3);
+  result.appendChild(resultInner);
+}
```

Comment:
- [ ] [required] It has error handling for promises or async/await ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/error_handling_requirement.png) from Pathwright) 
=> That would fix this issue on the screenshot
![Screenshot from 2021-05-12 13-23-41](https://user-images.githubusercontent.com/56886064/117967408-4a800d80-b325-11eb-87d5-a4c9f8cc9143.png)
=> That is the error which appears upon searching a city which doesn't exist